### PR TITLE
Fix admin

### PR DESCRIPTION
**Pull request by dfaulken**
Ref: https://github.com/umts/st-pax/pull/5/commits/9bd1f17ac19d14fb170c5bf941e0d888566e5939

Also took care of a few things I ran across while getting an app instance up and running.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -48,6 +48,7 @@ group :development do
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
   gem 'web-console', '>= 3.3.0'
   gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'rb-fsevent', '0.9.8'
```
Comment (dfaulken) [inline Gemfile]: https://github.com/guard/listen/issues/431

Comment (werebus) [inline Gemfile]: (Which appears to be fixed in rb-fsevent 0.10.2)

Comment (Chauce) [inline Gemfile]: Ah, I was wondering what was up with that!

Comment (Anbranin) [inline Gemfile]: Why don't we update rb-fsevent then?

Comment (dfaulken) [inline Gemfile]: Sure, go nuts.

**Comment by dfaulken** [comment]
Cool. Merge at your leisure.

**Pull request merged**